docker-compose.yml

```
@@ -0,0 +1,79 @@
+version: '3'
+
+services:
+  sources-db:
+    container_name: sources_db
+    image: postgres:12
+    environment:
+    - POSTGRES_DB=sources_api_development
+    - POSTGRES_USER=postgres
+    - POSTGRES_PASSWORD=postgres
+    ports:
+      - 15436:5432
+    volumes:
+      - ./tmp/db:/var/lib/postgresql/data
+
+  zookeeper:
+    container_name: zookeeper
+    image: confluentinc/cp-zookeeper
+    environment:
+      - ZOOKEEPER_CLIENT_PORT=32181
+      - ZOOKEEPER_SERVER_ID=1
+
+  kafka:
+    container_name: kafka
+    image: confluentinc/cp-kafka
+    ports:
+      - 9092:9092
+      - 29092:29092
+    depends_on:
+      - zookeeper
+    environment:
+      - KAFKA_ADVERTISED_LISTENERS=DOCKER://${KAFKA_QUEUE_HOST-kafka}:${KAFKA_QUEUE_PORT-29092},LOCALHOST://localhost:${KAFKA_QUEUE_PORT-9092}
+      - KAFKA_LISTENER_SECURITY_PROTOCOL_MAP=DOCKER:PLAINTEXT,LOCALHOST:PLAINTEXT
+      - KAFKA_INTER_BROKER_LISTENER_NAME=DOCKER
+      - KAFKA_BROKER_ID=1
+      - KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR=1
+      - KAFKA_ZOOKEEPER_CONNECT=zookeeper:32181
+      - KAFKA_AUTO_CREATE_TOPICS_ENABLE=true
+
+  sources-server:
+    container_name: sources_server
+    build:
+      context: .
+      dockerfile: Dockerfile
+    command: ["run_rails_server"]
+    entrypoint: ["entrypoint", "seed_database"]
```

Since we're overriding the `CMD` in the Dockerfile anyway (compose's `entrypoint` does it implicitly) - can we just remove it here and make the entrypoint:
```suggestion
    entrypoint: ["entrypoint", "seed_database", "run_rails_server"]
```